packages/enzyme-test-suite/test/selector-spec.jsx

```diff
@@ -21,8 +21,16 @@ const tests = [
   },
 ];
 
+let expectAttributeMatch;
+
 describe('selectors', () => {
   tests.forEach(({ describeMethod, name, renderMethod }) => {
+    before(() => {
```

Comment:
this is "before all"; could we use `beforeEach` instead?

Reply:
We could, but "before all" is the intention here, since `expectAttributeMatch` only needs to be initialized once per `renderMethod`

Reply:
fair; beforeAll tends to cause test flakiness in my experience